app/models/organization.rb

```diff
@@ -310,4 +342,11 @@ def set_ambassador_organization_defaults
     self.ascend_name = nil
     self.parent_organization_id = nil
   end
+
+  def not_both_regional_organization_and_suborganization
+    if regional? && regional_organization_id.present?
+      errors.add(:regional, :cannot_be_regional_and_belong_to_one)
+      errors.add(:regional_organization, :cannot_be_regional_and_belong_to_one)
```

Comment:
I don't think you need to add this error to both? I think adding to both will make the error message more confusing.

I think it makes more sense to just add it to `regional_organization`.